src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -92,10 +92,20 @@ export default class AlterationEnrichmentContainer extends React.Component<
             this.data,
             this._enrichedGroups,
             this.significanceFilter,
-            this.selectedGenes
+            this.filterFunction
         );
     }
 
+    @autobind
+    private filterFunction(hugoGeneSymbol: string) {
```

Comment:
should this be named filterByGene?  i mean, it's not generic so why not be more specific